src/configuration/option-source.ts

```diff
@@ -0,0 +1,8 @@
+enum OptionSource {
+    Configuration = 'Configuration',
+    Input = 'Input'
```

Comment:
Is it necessary to change the first char to upper case?

Reply:
I've used upper case naming notation as in TypeScript documentation - https://www.typescriptlang.org/docs/handbook/enums.html